docs/sdk_performance.md

```diff
@@ -0,0 +1,57 @@
+# SDK Performance
+
+## SDK Metrics
```

Comment:
This table doesn't make sense. You're mixing informations that don't belong together. I'd rather have a list of bullet points, with the key information in bold and then the concise explanation on how we gathered the data.